config/nav.yml

```diff
@@ -251,6 +257,13 @@ nav:
     # Eventing reference docs
       - Reference:
         - Eventing API: eventing/reference/eventing-api.md
+###############################################################################
+    # Functions
+###############################################################################
+    - Functions:
+      - About Knative Functions: functions/README.md
+      - Function Runtimes: functions/runtimes.md
+      - Local Development and Testing: functions/local-development.md
```

Comment:
I feel like this should be a separate PR since we have another issue for it: https://github.com/knative/docs/issues/5265

Reply:
Honestly, I would rather keep it all as one and we can follow up with updates if this is not sufficient. Any reason why one PR shouldn't close two issues?